modules/microshift-low-latency-config-yaml.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-config-yaml_{context}"]
+= Configuration kubelet parameters and values in {microshift-short}
+
+The first step in enabling low latency to a {microshift-short} cluster is to add configurations to the {microshift-short} `config.yaml` file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* You made a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+
+.Procedure
+
+. Add the kubelet configuration to the {microshift-short} `config.yaml` file:
++
+.Example passthrough `kubelet` configuration
+[source,yaml]
+----
+# ...
+kubelet: <1>
+  cpuManagerPolicy: static # <2>
+  cpuManagerPolicyOptions:
+    full-pcpus-only: "true" # <3>
+  cpuManagerReconcilePeriod: 5s
+  memoryManagerPolicy: Static # <4>
+  topologyManagerPolicy: single-numa-node
+  reservedSystemCPUs: 0-1 # <5>
+  reservedMemory:
+  - limits:
+      memory: 1100Mi # <6>
+    numaNode: 0
+  kubeReserved:
+    memory: 500Mi
+  systemReserved:
+    memory: 500Mi
+  evictionHard: # <7>
+    imagefs.available: "15%" # <8>
+    memory.available: "100Mi" # <9>
+    nodefs.available: "10%" # <10>
+    nodefs.inodesFree: "5%" # <11>
+  evictionPressureTransitionPeriod: 0s
+# ...
+----
+<1> If you change the CPU or memory managers in the kubelet configuration, you must remove files that cache the previous configuration. Restart the host to remove them automatically, or manually remove the `/var/lib/kubelet/cpu_manager_state` and `/var/lib/kubelet/memory_manager_state` files.
+<2> The name of the policy to use. Valid values are `none` and `static`. Requires the `CPUManager` feature gate to be enabled. Default value is `none`.
```

Comment:
>Requires the `CPUManager` feature gate to be enabled.

Should this be a prereq? The wording might be tricky, as it seems required for only a couple of parameters. But, it might prevent the user from stopping cold here and needing to go back and do something else. (I'd get annoyed!).

Reply:
Hmmm. @pmtk Should this note be here, or is this referring to an OCP CR we don't use?